.prospector.yaml

```diff
@@ -1,14 +1,58 @@
 output-format: grouped
-inherits:
-  - full_pep8
-  - strictness_low
+strictness: veryhigh
+test-warnings: true
+doc-warnings: true
+max-line-length: 79
 pep8:
-  options:
-    max-line-length: 79
+  full: true
 pylint:
   enable:
     - relative-import
+  disable:
+    - C0111  # Missing %s docstring
+    - R0201  # Method could be a function
```

Comment:
I like this one as well but it triggers very often in test methods. We could disable it in the test files specifically instead